test/Application.UTest/Common/Files/DtvDataTest.cs

```diff
@@ -8,14 +8,14 @@ public class DtvDataTest
     [Test]
     public void CheckBotExists()
     {
-        string classDivisionsXmlPath = Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory) + "/ModuleData/class_divisions.xml";
+        string classDivisionsXmlPath = Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory) + "/ModuleData/dtv/dtv_class_divisions.xml";
         XDocument classDivisionsDoc = XDocument.Load(classDivisionsXmlPath);
         var classDivisionIds = classDivisionsDoc
             .Descendants("MPClassDivision")
             .Select(el => el.Attribute("id")!.Value)
             .ToHashSet();
 
-        string dtvDataXmlPath = Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory) + "/ModuleData/dtv_data.xml";

```

Comment:
What about regular bots?

Reply:
We should not reference regular bots in dtv_data.xml. As this test is only checking for the existence of class_divisions referenced in dtv_data, I think this should be fine to only check for the dtv class_divisions.

Reply:
My nad yo'ure right